src/main/webapp/app/account/password-reset/init/password-reset-init.component.html

```diff
@@ -1,37 +1,16 @@
-<div *ngIf="passwordResetEnabled">
```

Comment:
Did something go wrong there with some merge conflicts? Because this is my change that is already in develop some time.

Reply:
Yeah, weird.